dev-docs/bidders/silverpush.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: bidder
+title: Silverpush
+description: Prebid Silverpush Bidder Adaptor
+pbjs: true
+pbs: true
+biddercode: silverpush
+media_types: banner, video
+schain_supported: true
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
+floors_supported: true
+userIds: 
+prebid_member: false
+fpd_supported: false
+multiformat_supported: will-bid-on-one
```

Comment:
awesome!

You may also add the `safeframes_ok`: `true` or `false` to make setup even faster